packages/react-components/react-theme/src/index.ts

```diff
@@ -1,9 +1,9 @@
-export { teamsDarkTheme, teamsHighContrastTheme, teamsLightTheme, webDarkTheme, webLightTheme } from './themes/index';
-export { createDarkTheme, createHighContrastTheme, createLightTheme, createTeamsDarkTheme } from './utils/index';
+export { teamsDarkTheme, teamsHighContrastTheme, teamsLightTheme, webDarkTheme, webLightTheme } from '@fluentui/tokens';
```

Comment:
I'm fine if we keep the exports this way because they are logically grouped, but should these exports be updated into a single `export` since they are all `from '@fluentui/tokens'` now?